### PR TITLE
common.mk post-build.sh S30optee: change /data to /var/lib

### PR DESCRIPTION
**Pull request by mikkorapeli-linaro**
/data/tee is not FHS compatible path. Use /var/lib/tee instead. Related to optee_client side CMake change to use standard CMake install and runtime paths:
https://github.com/OP-TEE/optee_client/pull/391

### CODE REVIEW

**Comment by mikkorapeli-linaro** [comment]
Is there a way to mark this PR to depend on https://github.com/OP-TEE/optee_client/pull/391 to test both together?

**Comment by jforissier** [comment]
> Is there a way to mark this PR to depend on [OP-TEE/optee_client#391](https://github.com/OP-TEE/optee_client/pull/391) to test both together?

No, mentioning it in plain English as you did is the best you can do :) thanks!

**Review by jforissier — APPROVED** [review]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by jforissier** [comment]
Merged, thanks.
